add admin questions (#162)
FidelusAleksander/ghcertified: content/questions/admin/question-006.md

---

> question: "Which of the following is the correct procedure for generating a support bundle in GitHub Enterprise Server?"

Reviewer: ```suggestion
question: "Which of the following is the correct procedure for generating a support bundle in GitHub Enterprise Server? (Choose two.)"
```

> question: "Which of the following are correct methods to generate and share a diagnostic file for GitHub Enterprise Server?"

Reviewer: ```suggestion
question: "Which from the following is a correct method to generate and share a diagnostic file for GitHub Enterprise Server?"
```

archetype: "questions"
title: "Question 006"
draft: false
---

> 

Reviewer: Any documentation link  to this one?

Author: Nope sadly.

Reviewer 2: This has some documentation about the tool `ghe-diagnostics` : https://docs.github.com/en/enterprise-server@3.10/support/contacting-github-support/providing-data-to-github-support#creating-and-sharing-diagnostic-files

Author: @chrheg  Thanks but I had to change this since it was a duplicate of Q5.
